source/hassio/debugging.markdown

```diff
@@ -13,7 +13,7 @@ The following debug tips and tricks are for people who are running the Hass.io i
 
 ## {% linkable_title SSH access to the host %}
 
-Create an `authorized_keys` file in the root of your SD card with your public key. Once the device is booted, you can access your device as root over SSH on port 22222.
+Create an `authorized_keys` file in the root of your SD card with your public key. Once the device is booted, you can access your device as root over SSH on port 22222. The contents of the 'authorized_keys' file should be the contents of your 'id_rsa.pub' file.
```

Comment:
That's true but I'm just a tad unsure if you really should specify the file name since it's not necessarily correct?